Sources/Segment/Utilities/Storage.swift

```diff
@@ -207,7 +207,8 @@ extension Storage {
         #if os(tvOS) || os(macOS) || os(Windows)
         let searchPathDirectory = FileManager.SearchPathDirectory.cachesDirectory
         #else
-        let searchPathDirectory = FileManager.SearchPathDirectory.documentDirectory
+        // Do not use `documentDirectory` on iOS or they can appear in the Files app
+        let searchPathDirectory = FileManager.SearchPathDirectory.cachesDirectory
```

Comment:
Should we drop this if/else entirely (both branches are the same now)?

Also, any potential issues from dropping whatever was stored in the documents directory?

Reply:
There is a chance that after the app update any events that were not sync'd would be missed. I could write a migration path, which actually would be nice so we can cleanup the segment folder from existing users so its not shown in files